BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java

```diff
@@ -197,10 +201,14 @@ public TaskState complete(long taskId){
 
 	public Set<Task> getTasksStartedLaterThan(Date start, String name) {
 		String pattern = "yyyy-MM-dd";
+		Calendar c = Calendar.getInstance();
+		c.setTime(start);
+		c.add(Calendar.DATE, 1);
 		SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
-		java.sql.Date qdate = new java.sql.Date(start.getTime());
+		Date sDate = new Date(start.getTime());
```

Comment:
Looks like it is unnecessary to create this instance: we can use `start` directly

Reply:
good catch removed.